content/changelog/2025-03-28.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Neon serverless driver GA, easier Neon Auth setup, MCP Server updates, and more
+---
+
+## Neon serverless driver is now generally available (GA)
+
+The [Neon serverless driver](https://github.com/neondatabase/serverless) for JavaScript/TypeScript has reached **version 1.0.0** and is now generally available! Built for environments like Vercel Functions, Cloudflare Workers, and even browsers, the driver carries SQL over HTTP or WebSockets — no raw TCP required.
+
+This GA release brings a cleaner, more maintainable codebase, stronger SQL injection safeguards, support for composable tagged-template queries, and better performance when inserting binary data over HTTP.
+
+**What you need to know about the 1.0.0 GA release:**
+
+- It requires **Node.js v19 or later**.
+- It includes a **breaking change** but only if you're calling the HTTP query template function as a conventional function. The first usage shown below remains safe and supported. However, the second usage is an SQL injection risk (notice the parentheses) and is no longer permitted — it now throws an error. You'll need to update your app if you use it. 
+
+    ```javascript
+    // this usage remains safe and supported
+    const resultA = await sql`SELECT * FROM table WHERE id = ${id}`;
+    // this usage is not safe and now throws an error
+    const resultB = await sql(`SELECT * FROM table WHERE id = ${id}`);
+    ```
+
+For more about these changes and others in the 1.0.0 GA release, please see the [1.0.0 release notes](https://github.com/neondatabase/serverless/pull/149) or read the [blog post](https://neon.tech/blog/serverless-driver-ga).
+
+## Streamlined Neon Auth setup
+
+We've simplified Neon Auth onboarding so you can add authentication to your project faster. With a cleaner UI and clearer steps, it's now quicker to sync users directly to your database. Get started adding your first users with Neon Auth.
+
+![Streamlined Neon Auth setup](/docs/relnotes/neon_auth_splash.png)
+
+Learn more in our [docs](https://neon.tech/docs/guides/neon-auth).
+
+## MCP Server updates
+
+We're continuing to improve our MCP Server. This week, we added built-in support for the [AI rules](https://github.com/neondatabase-labs/ai-rules) we announced two weeks ago: Neon Auth, our serverless driver, and Drizzle resources.
```

Comment:
```suggestion
We're continuing to improve our MCP Server. This week, we added built-in support for the [AI rules](https://github.com/neondatabase-labs/ai-rules) we announced two weeks ago: Neon Auth, our serverless driver, and Drizzle resources.
```
```suggestion
We're continuing to improve our MCP Server. This week, we added built-in support for the [AI rules](https://github.com/neondatabase-labs/ai-rules) we announced two weeks ago. In case you missed it, we  provide AI rules for setting up Neon Auth, querying your database with the Neon serverless driver, and integrating Neon with Drizzle ORM. Keep an eye out for more AI rules as we expand this resource.
```